tf2onnx/onnx_opset/nn.py

```diff
@@ -587,67 +587,6 @@ def version_11(cls, ctx, node, **kwargs):
         cls.version_1(ctx, node, **kwargs)
 
 
-@tf_op(["CropAndResize"])
```

Comment:
tf2onnx does not depend on onnxruntime or contrib ops. You need to check if the ms namespace is enabled.

Reply:
Sure, but how to verify if the namespace enabled?

Reply:
So by default we use onnx opset. Than we override with the extra-opsets.
So you'd add it to ms.py but keep the original code in controflow.py for runtimes that don't support the ms namespace.
For example the Range op:
https://github.com/onnx/tensorflow-onnx/blob/master/tf2onnx/onnx_opset/controlflow.py#L256
https://github.com/onnx/tensorflow-onnx/blob/master/tf2onnx/custom_opsets/ms.py#L32

Reply:
@guschmue, restored nn.py to keep assembled one.